### ISSUE TITLE
`gopass mv` decrypts the entry

### ISSUE THREAD

**Issue by AnomalRoil** [gopasspw/gopass#2181]
### Summary
When running `gopass mv a/path/to/entry b/new/path/name`, Gopass wants to decrypt the entry, which doesn't really make sense since it doesn't need to know its content to move it around on the filesystem. 

### Steps To Reproduce
Run gopass without the private key for that store. 
(I do it by unplugging my Yubikey, but there are other ways to achieve it by setting `GNUPGHOME=some/path` e.g.)

```
$ gopass generate test/to/move 20 
✅ Password for entry "test/to/move" generated

$ gopass mv test/to/move test/new/path
gpg: decryption failed: No secret key
❌ Decryption failed: exit status 2

Error: source test/to/move does not exist in source store : failed to decrypt
```

Plus notice the error message is quite cryptic and factually wrong since it says "source test/to/move does not exist in source store".

### Expected behavior

This should work without decrypting the entry first.

### Environment

- OS version: based on Ubuntu 21.10, kernel  `5.16.15-76051615-generic x86_64 GNU/Linux`
- gopass Version: `gopass 1.14.0-git+HEAD`
- Installation method: `go install`

**Comment by dominikschulz** [comment]
Yes, that's a shortcut I took in the implementation.

But it shouldn't be too hard to avoid decryption

**Comment by dominikschulz** [comment]
Back when we did support non-FS based backends I did generalize the [move implementation](https://github.com/gopasspw/gopass/blob/master/internal/store/root/move.go#L152) to always Get&Set.

But now only support FS-based backends in practice so I should be worth changing that.
We'll still need a fallback to the old implementation, but that should be easy.

Will try to look into this soon, as it's an important usability issue (I've been unhappy about this for a while but did never bother to do anything).
